spec/cell_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'cell'
+
+describe Cell do
+
+  let(:cell){ Cell.new }
+
+  describe '#new' do
+    it 'is initialized without a mine' do
+      expect(cell.mine).to be false
+    end
+    it 'is initialized as uncleared' do
+      expect(cell.cleared).to be false
+    end
+    it 'is initialized without a flag' do
+      expect(cell.flagged).to be false
+    end
+    it 'is initialized with adjacent_mines equal to zero' do
+      expect(cell.adjacent_mines).to be 0
```

Comment:
Originally we had adjacent_mines as nil.... Eventually we realized how much better our lives would be if we initialized at 0.